Rubberduck.Core/Navigation/CodeExplorer/CodeExplorerComponentViewModel.cs

```diff
@@ -68,8 +68,9 @@ public CodeExplorerComponentViewModel(CodeExplorerItemViewModel parent, Declarat
                         var component = _projectsProvider.Component(qualifiedModuleName);
                         string parenthesizedName;
                         using (var properties = component.Properties)
+                        using (var nameProperty = properties["Name"])
                         {
-                            parenthesizedName = properties["Name"].Value.ToString() ?? string.Empty;
+                            parenthesizedName = nameProperty.Value.ToString() ?? string.Empty;
```

Comment:
Missing in the original code too, but this could use a null check on `nameProperty`.

Reply:
I'm not sure I follow this. Wouldn't it have had thrown on the `var nameProperty = properties["Name"]` if it didn't exist? I don't think it's possible for a `IProperties` to contain a null property that's named? 

In fact, R# flags the `?? string.Empty` as redundant since `ToString()` can never be `null` apparently. I changed it to `nameProperty.Value?.ToString() ?? string.Empty;` because `Value` is an `object` and could be `null`. That made R# happy. Is what you meant?